mysite/urls.py

```diff
@@ -14,8 +14,9 @@
     2. Add a URL to urlpatterns:  path('blog/', include('blog.urls'))
 """
 from django.contrib import admin
-from django.urls import path
+from django.urls import path, include
 
 urlpatterns = [
-    path('admin/', admin.site.urls),
+    path('summernote/', include('django_summernote.urls')),
```

Comment:
this is required when using summer note widget as a field because widget class uses a template View to put editor in iframe and to call template view it uses reverse, hence including summernote.urls is required.